### PR TITLE
MEN-4711

### PR DESCRIPTION
**Pull request by mzedel**

### CODE REVIEW

**Comment by tranchitella** [comment]
@mzedel there is a spurious "tooltip" key in the update config map you send to the backend:
image

**Comment by tranchitella** [comment]
@mzedel clicking on the "view details" link for a deployment produces:
image

**Comment by tranchitella** [comment]
Audit logs, as well, are not working properly:
image

**Comment by mzedel** [comment]
@tranchitella these issues should be addressed with #1724 and mendersoftware/integration#1425 - as a workaround, starting the env with `-f docker-compose.auditlogs.yml` should be enough

**Comment by tranchitella** [comment]
@mzedel what about the config map containing the tooltip attribute?

**Comment by mzedel** [comment]
that is a left over from the step definitions from the UI and during the next round of edits I'll add a step to filter out any additional fields [here](https://github.com/mendersoftware/gui/pull/1696/files#diff-adfa51522334220857578b3356028691c6fee975bfd105fcf44a532516d6cbbfR23)

File: src/js/components/deployments/deployment-report/phaseprogress.js [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    failureIndicators: deploymentStatesToSubstates.failures
+  },
+  install: {
+    title: 'install',
+    done: 'installed',
+    successIndicators: ['rebooting', 'pause_before_rebooting', 'pause_before_committing', 'success'],
+    failureIndicators: []
+  },
+  reboot: { title: 'reboot', done: 'rebooted', successIndicators: ['pause_before_committing', 'success'], failureIndicators: [] },
+  commit: { title: 'commit', done: 'comitted', successIndicators: deploymentStatesToSubstates.successes, failureIndicators: [] }
```
Comment (mzedel) [inline src/js/components/deployments/deployment-report/phaseprogress.js:32]: @kacf here it is! - I'll fix it once I get to the next round of changes

Comment (kacf) [reply src/js/components/deployments/deployment-report/phaseprogress.js:32]: Aha, it is lowercase! That explains it.

**Review by tranchitella — COMMENTED** [review]
@mzedel I cannot find the button to continue on pause, see screenshot:

image

**Comment by mzedel** [comment]
Afaik that's still waiting for the fixed statistic reporting in the client...

**Review by tranchitella — APPROVED** [review]
Works fine!

**Pull request merged**